src/pipx/commands/inject.py

```diff
@@ -11,6 +13,10 @@
 from pipx.util import PipxError
 from pipx.venv import Venv
 
+logger = logging.getLogger(__name__)
+
+COMMENT_RE = re.compile(r"(^|\s+)#.*$")
```

Comment:
I'd inline that?

Reply:
Do you mean move it to `parse_requirements`? It's used in a loop, so it's worth compiling (I assume). Also, this is how it's implemented in https://github.com/pypa/pip/blob/main/src/pip/_internal/req/req_file.py.